<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GridShapeSettingEditWidget</class>
 <widget class="QWidget" name="GridShapeSettingEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>186</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="linesGroupBox">
     <property name="title">
      <string>Grid Lines</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="allRadioButton">
        <property name="text">
         <string>All</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="outlineRadioButton">
        <property name="text">
         <string>Outline Only</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="gridColorLabel">
          <property name="text">
           <string>Color:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="ColorEditWidget" name="gridColorEditWidget">
          <property name="baseSize">
           <size>
            <width>100</width>
            <height>50</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="indicesGroupBox">
     <property name="title">
      <string>Grid Indices</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="indexVisibleCheckBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Visible</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="vtkTextPropertySettingWidget" name="indexTextSettingWidget" native="true">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorEditWidget</class>
   <extends>QLabel</extends>
   <header>guibase/widget/coloreditwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>vtkTextPropertySettingWidget</class>
   <extends>QWidget</extends>
   <header>vtktextpropertysettingwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>allRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>indexVisibleCheckBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>27</x>
     <y>37</y>
    </hint>
    <hint type="destinationlabel">
     <x>26</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>indexVisibleCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>indexTextSettingWidget</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>54</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
